modules/dashboard/test/DashboardTest.php

```diff
@@ -36,10 +53,165 @@ class DashboardTest extends LorisIntegrationTest
+        $this->assertContains("View site breakdown", $assertText2);
+    }
+    /**
+  * Verify that for a user with 'conflict_resolver' permission,
+  * Check that site displayed is always 'All'.
+  * Click on this task and verify that you go to the conflict_resolver page.
+  *
+  *@return void
+  */
+    public function testConflictResolverPermission()
+    {
+        // check the element which shows on the My tasks panel
+         $this->setupPermissions(array("conflict_resolver"));
+             // check the link
+             $this->safeGet($this->url . "/conflict_resolver/");
+             $bodyText = $this->safeFindElement(WebDriverBy::cssSelector("body"))
+                 ->getText();
+
+             $this->assertNotContains(
+                 "You do not have access to this page",
+                 $bodyText
+             );
+             $this->resetPermissions();
+
+    }
+    /**
+   * Verify that for a user with 'Violated Scans: View all-sites' permissions,
+   * Check that site displayed is always 'All'.
+   *
+   * @return void
+   */
+    public function testViolatedPermission()
+    {
+         $this->setupPermissions(array("violated_scans_view_allsites"));
+         // check the link
+         $this->safeGet($this->url . "/mri_violations/");
+         $bodyText = $this->safeFindElement(WebDriverBy::cssSelector("body"))
+             ->getText();
+         $this->assertNotContains("You do not have access to this page", $bodyText);
+         $this->resetPermissions();
+
+    }
+    /**
+  * Verify that for a user with 'Across all sites create and edit user
+  * accounts' permission.
+  * Check that site displayed is always 'All'.
+  *
+  * @return void
+  */
+    public function testAcrossAllPermission()
+    {
+         $this->setupPermissions(array("user_accounts_multisite", "user_accounts"));
+         // check the link
+         $this->safeGet($this->url . "/user_accounts/");
+         $bodyText = $this->safeFindElement(WebDriverBy::cssSelector("body"))
+             ->getText();
+         $this->assertNotContains("You do not have access to this page", $bodyText);
+         $this->resetPermissions();
+
+    }
+    /**
+   * Verify that for a user with "edit Final radiological review" or
+   * "view_final_radiological_review" permission.
+   * Check that site displayed is always 'All'.
+  *
+   * @return void
+   */
+    public function testRadiologicalPermission()
+    {
+         $this->setupPermissions(
+             array(
+              "edit_final_radiological_review",
+              "view_final_radiological_review",
+             )
+         );
+         // check the link
+         $this->safeGet($this->url . "/final_radiological_review/");
+         $bodyText = $this->safeFindElement(WebDriverBy::cssSelector("body"))
+             ->getText();
+         $this->assertNotContains("You do not have access to this page", $bodyText);
+         $this->resetPermissions();
+
```

Comment:
The above tests all look like they have the same problem as the conflict_resolver test.